torch/ao/quantization/fx/backend_config/native.py

```diff
@@ -78,6 +78,16 @@ def _get_default_op_backend_config(op, dtype_configs):
         torch.nn.functional.layer_norm,
     ]]
 
+_ADD_CONFIG = {
+    "pattern": operator.add,
+    "observation_type": ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT,
+    "_is_binary_op_with_binary_scalar_op_variant": True,
```

Comment:
Thoughts on why there is an underscore?  It's ok to make this public.

Reply:
just feel this is temporay and we may have more general api after the dtype inference support

Reply:
this is changed to a map from num_tensor_args to the observation type